### ISSUE TITLE
"error while reading: UnexpectedEOF" reading MultiLineString

### ISSUE THREAD

**Issue by pka** [andelf/rust-postgis#6]
I get this exception for MultiLineString (and also MultiPolygon) when reading any data in EPSG:3857.

```
println!(">>>>>> {:#?}", row.get::<_, postgis::MultiLineString<postgis::Point<EPSG_3857>>>("wkb_geometry"));
```

->
`thread 'datasource::postgis::test_from_geom_fields' panicked at 'called "Result::unwrap()" on an "Err" value: Read("error while reading: UnexpectedEOF")', ../src/libcore/result.rs:688`

Here's a failing MultiLineString:
`'0105000020110F000001000000010200000004000000A149950DA3BB564189DF62973DC35C4165CD9E8413C6564189DF62973DC35C410C6A639A84C856413CC52F806BC25C4159E55A3637CD5641D6FD917DC6BE5C41'::geometry`
`SRID=3857;MULTILINESTRING((5959308.21223679 7539958.36540974,5969998.07219252 7539958.36540974,5972498.41231776 7539118.00291568,5977308.84929784 7535385.96203562))`

**Comment by andelf** [comment]
Thanks for your reporting. Before this to be fix, try have a look at https://github.com/georust .:)

**Comment by pka** [comment]
Fixed with PR #9
